apps/modernization-ui/src/assets/shared.scss

```diff
@@ -19,3 +19,7 @@ a {
         text-decoration: underline;
     }
 }
+
+.usa-button {
```

Comment:
issue: This is changing every button, checkout out Advanced Search.  What I meant by the previous comment that the `add-patient-button` exited in the Advanced Search styles but was used in another page.  This signals to me that this class is a global style that should be available to the entire application so it should exist in the `share.scss` file.

question: What is the reason for the `margin: 0`?

Reply:
Yes so those styles are used in advanceSearch and in the addPatient page, I will move both those styles to the shared.scss instead of AdvancedSearch.scss. 

margin: 0 is needed to remove spaces from top and bottom to make it horizontally and vertically aligned as it has margin from the uswds class so Im overriding it using margin: 0